src/vector_index.h

```diff
@@ -41,6 +43,25 @@
 #define VECSIM_TYPE "TYPE"
 #define VECSIM_DIM "DIM"
 #define VECSIM_DISTANCE_METRIC "DISTANCE_METRIC"
+#define VECSIM_GRAPH_DEGREE "GRAPH_MAX_DEGREE"
+#define VECSIM_WINDOW_SIZE "CONSTRUCTION_WINDOW_SIZE"
+#define VECSIM_NUM_THREADS "NUM_THREADS"
+#define VECSIM_WSSEARCH "SEARCH_WINDOW_SIZE"
+#define VECSIM_MAX_CANDIDATE_POOL_SIZE "MAX_CANDIDATE_POOL_SIZE"
+#define VECSIM_PRUNE_TO "PRUNE_TO"
+#define VECSIM_ALPHA "ALPHA"
+#define VECSIM_USE_SEARCH_HISTORY "USE_SEARCH_HISTORY"
+#define VECSIM_USE_SEARCH_HISTORY_ON "ON"
+#define VECSIM_USE_SEARCH_HISTORY_OFF "OFF"
+#define VECSIM_USE_SEARCH_HISTORY_DEFAULT "DEFAULT"
+#define VECSIM_QUANT_BITS "COMPRESSION"
+#define VECSIM_LVQ_4 "4"
+#define VECSIM_LVQ_8 "8"
+#define VECSIM_LVQ_4X4 "4X4"
+#define VECSIM_LVQ_4X8 "4X8"
```

Comment:
Note that `VECSIM_QUANT_BITS` represents `COMPRESSION` now, so `VECSIM_LVQ_x` variants should be `LVQx` (not only `x`)